source/jormungandr/jormungandr/interfaces/v1/fields.py

```diff
@@ -243,6 +243,12 @@ def output(self, key, obj):
             r.append({"id": note_.uri, "value": note_.note})
         return r
 
+class realtime_level(fields.Raw):
```

Comment:
I don't think you need this, there is already some generic code to output some enum